fdbserver/MasterProxyServer.actor.cpp

```diff
@@ -258,15 +261,18 @@ ACTOR Future<Void> getRate(UID myID, Reference<AsyncVar<ServerDBInfo>> db, int64
 		when ( wait( nextRequestTimer ) ) {
 			nextRequestTimer = Never();
 			bool detailed = now() - lastDetailedReply > SERVER_KNOBS->DETAILED_METRIC_UPDATE_RATE;
-			
+
 			TransactionTagMap<uint64_t> tagCounts;
 			for(auto itr : *throttledTags) {
 				for(auto priorityThrottles : itr.second) {
 					tagCounts[priorityThrottles.first] = (*transactionTagCounter)[priorityThrottles.first];
 				}
 			}
-			reply = brokenPromiseToNever(db->get().ratekeeper.get().getRateInfo.getReply(GetRateInfoRequest(myID, *inTransactionCount, *inBatchTransactionCount, tagCounts, detailed)));
+			reply = brokenPromiseToNever(db->get().ratekeeper.get().getRateInfo.getReply(
+			    GetRateInfoRequest(myID, *inTransactionCount, *inBatchTransactionCount, *transactionTagCounter,
```

Comment:
@sfc-gh-xwang It looks like this is the change that reverted the use of `tagCounts` introduced in my other PR. Seems like a merge issue because we both edited the same line.

Reply:
Sorry about it. It seems like `transactionTagCounter` and `tagCounts` are mixed by mistake. I'd fix it in a new PR.